channels/packages/metrics-exporter/0.1.1/daemonset-storageos-metrics-exporter.yaml

```diff
@@ -0,0 +1,46 @@
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: storageos
+  name: storageos-metrics-exporter
+  namespace: storageos
+spec:
+  selector:
+    matchLabels:
+      app: storageos
+  template:
+    metadata:
+      labels:
+        app: storageos
+        category: metrics-exporter
+    spec:
+      containers:
+      - image: storageos/metrics-exporter:v0.1.1
+        imagePullPolicy: IfNotPresent
+        name: storageos-metrics-exporter
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /var/lib/storageos
+          name: state
+          readOnly: true
+        - mountPath: /var/lib/kubelet
+          mountPropagation: Bidirectional
```

Comment:
I remember us discussing this but I can't remember the outcome, is `HostToContainer` sufficient enough or do we definitely need `Bidirectional` mount propagation?

Reply:
Definitely need the `Bidirectional` here otherwise new mounts won't be accessible by the metrics-exporter pod until it is restarted